### ISSUE TITLE
PBS example

### ISSUE THREAD

**Issue by bkmgit** [carpentries-incubator/hpc-intro#366]
It may be helpful to have drop down options for choosing the scheduler (SLURM vs PBS). This makes the lesson a reference for many different sites. No need to fork repository for different site specific lessons, this will enable more community updates in the main branch rather than in forks.

**Comment by Anselm-UTS** [comment]
Having a PBS as a alternative would be helpful, but it might cause confusion for during the lesson if users are looking or tying the PBS commands while the SLURM commands are being taught.

**Comment by bkmgit** [comment]
Yes, confusion should be avoided. Usually expect instructor to explain which system is being used, instructor notes and lesson plan should make it clear which one is being used.  It is likely that HPC users will need to use both PBS and SLURM since they will typically use several different resources over time.

**Comment by reid-a** [comment]
The snippet library is meant to take care of this, but perhaps is not meeting the need?  The idea is that, when building the lesson for the workshop, you enable the appropriate snippet library, and then conflicting info does not appear.

**Comment by reid-a** [comment]
We have talked a bit about having generic SLURM and generic PBS configurations, with site-specific snippets being in a tier below that, but it's not obvious that the benefit outweighs the additional complexity of the snippets that this implies. Also it's not obvious to me how many learners actually attend workshops vs. grabbing info off the website -- in the latter case, they may not be aware of the snippets, and certainly they will not have been configured for a workshop.
